### ISSUE TITLE
Add logging into each "catch" of a try catch in the classes.

### ISSUE THREAD

**Issue by cleamws** [WorldShippingInc/EDIService#35]
Add logging into each "catch" of a try catch in the classes.

Add the below code to each one and make sure to change the function name being passed through on each one.

ex:

private void GETTHEDATA()
(
  Try
  {}
  catch(Exception ex)
  {
  className = MethodBase.GetCurrentMethod().DeclaringType.Name;
                Log.WriteToLog("E", className.ToString(), "GETTHEDATA", ex.Message.ToString(), sqlConnectionString);
  }
)

**Comment by clemwcp** [comment]
So there are some with comments like "do nothing and leave the string empty" Should I still add this? (Just checking)

**Comment by cleamws** [comment]
yeah, if they are in the Helper class, then leave them like that.  They are supposed to return nothing
